tests/evals/test_dataset.py

```diff
@@ -1496,10 +1496,36 @@ async def mock_async_task(inputs: TaskInput) -> TaskOutput:
                     'assertion_pass_rate': 1.0,
                     'logfire.msg_template': 'evaluate {name}',
                     'logfire.msg': 'evaluate mock_async_task',
+                    'experiment.metadata': {
+                        'n_cases': 2,
+                        'averages': {
+                            'name': 'Averages',
+                            'scores': {'confidence': 1.0},
+                            'labels': {},
+                            'metrics': {},
+                            'assertions': 1.0,
+                            'task_duration': 1.0,
+                            'total_duration': 9.0,
+                        },
+                    },
                     'logfire.span_type': 'span',
                     'logfire.json_schema': {
                         'type': 'object',
-                        'properties': {'name': {}, 'n_cases': {}, 'assertion_pass_rate': {}},
+                        'properties': {
+                            'name': {},
+                            'n_cases': {},
+                            'experiment.metadata': {
+                                'type': 'object',
+                                'properties': {
+                                    'averages': {
```

Comment:
I'd expect `n_cases` to also be listed here

Reply:
That's normal, schemas for primitives only appear at the top level.